deployments/mdcb/bootstrap.sh

```diff
@@ -50,20 +62,20 @@ set_docker_environment_value "MDCB_USER_API_CREDENTIALS" "$dashboard_mdcb_user_a
 log_ok
 bootstrap_progress
 
-# restart containers to use updated MDCB credentials
-log_message "Restarting MDCB deployment containers to use updated MDCB user API credentials"
-docker-compose \
-    -f deployments/tyk/docker-compose.yml \
-    -f deployments/mdcb/docker-compose.yml \
-    -p tyk-demo \
-    --project-directory $(pwd) \
-    up -d --no-deps --force-recreate tyk-mdcb tyk-worker-gateway 2> /dev/null
+# recreate containers to use updated MDCB credentials
+log_message "Recreating MDCB deployment containers to use updated MDCB user API credentials"
```

Comment:
It's because MDCB needs to use the API key of a user created during bootstrap, and because the value is set through an environment variable we need to restart the container so that it picks up the new value.